### ISSUE TITLE
Nether under overworld feature suggestion

### ISSUE THREAD

**Issue by florensie** [Johni0702/BetterPortals#276]
A world gen option that replaces the bottom bedrock layer and the top nether layer with portals for a terraria-style nether. Seems ambitious but it should be a lot better (easier and more compatibility) than actually generating the nether and the overworld in the same dimension.

**Comment by Johni0702** [comment]
More or less #57.

> should be a lot better (easier and more compatibility) than actually generating the nether and the overworld in the same dimension.

I'm highly skeptical of that. Even though general compatibility while you're no where close to the portal would be better, once you get close to it, lots of issues start popping up and nothing will "just work", i.e. everything needs to be manually supported (see #10, #44, #63, #74 and #117 for examples).

Given CubicChunks is a thing and works surprisingly well, the hard part would merely be having two world generators in the same world (which afaik should be fairly easy with CC's world gen) and then fixing up certain nether behavior (e.g. fog color, water evaporation, etc.) to happen based on y-coordinate rather than dimension.
